nltk/compat.py

```diff
@@ -544,6 +568,8 @@ def add_py3_data(path):
 
 # for use in adding /PY3 to the second (filename) argument
 # of the file pointers in data.py
+
+
 def py3_data(init_func):
```

Comment:
Comments belong to the function; it is better to either add a blank line above the comment or convert comments to a docstring